setup.py

```diff
@@ -20,7 +20,7 @@
     packages=find_packages(exclude=["tests*"]),
     include_package_data=True,
     zip_safe=False,
-    install_requires=["starlette>=0.12,<0.13", "tartiflette>=0.12,<0.13"],
+    install_requires=["starlette>=0.12,<0.13", "tartiflette>=0.12,<1.2"],
```

Comment:
Since the behavior of subscriptions is a breaking change, let's require users to use 1.0+:

```suggestion
    install_requires=["starlette>=0.12,<0.13", "tartiflette>=1.0,<1.2"],
```